src/parser.yy

```diff
@@ -411,6 +414,7 @@ expr_stmt:
  * this avoids a r/r conflict
  */
         |       assign_stmt        { $$ = $1; }
+        |       var_decl_stmt      { $$ = $1; }
```

Comment:
Is this needed here? It's not really an expression statement and doesn't seem to cause conflicts if I delete it. Maybe it should be a new top-level statement type? i.e. in the `stmt_list` rule

Reply:
> It's not really an expression statement

You're right it doesn't really produce a value which seems to be the requirement for an expression. 

> Maybe it should be a new top-level statement type? i.e. in the stmt_list rule

Yeah, that works. I'll move it there.